tests/integration/seccomp-notify.bats

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+	teardown_seccomphook
+	setup_seccomphook
+
+	teardown_seccompagent
+	setup_seccompagent
+
+	teardown_debian
+	setup_debian
```

Comment:
```suggestion
	teardown_seccompnotify
	setup_seccompnotify
```

And `s/debian/seccompnotify/g` in this file

Reply:
`setup_debian` is used to get a debian based container bundle. I changed it to busybox to make it faster but I can't rename it to `setup_seccompnotify`.